http_source.go

```diff
@@ -120,6 +122,16 @@ func replaceUrlTemplate(base string) string {
 		Ext:  ext,
 	}
 
+	exe, err := getExecutableRealPath()
+	if err != nil {
```

Comment:
Doesn't this belong in `getExecutableRealPath`?
Then if the error is returned we cannot ignore it...